### PR TITLE
Added semicolons

### PR DESCRIPTION
**Pull request by dhakerShiv**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-  return a + b
+  return a + b;
 }
 
 function multiply (a, b) {
-  return a * b
+  return a * b;
 }
 
 function subtract (a, b) {
```
Comment (dhakerShivnarayan) [inline index.js]: Add a semicolon in this function as well!

File: index.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 function add (a, b) {
-  return a + b
+  return a + b;
 }
 
 function multiply (a, b) {
-  return a * b
+  return a * b;
```
Comment (dhakerShivnarayan) [inline index.js]: That's a good thing, you are following eslint

Comment (dhakerShiv) [reply index.js]: Thanks!

File: index.js (hunk 1 above)
Comment (dhakerShiv) [reply index.js]: Added!

**Pull request merged**